participants/Novel-Wiki.md

```diff
@@ -0,0 +1,62 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+*Novel-Wiki,the goal is to create a wiki catalog of works of fiction.People can retrieve novels uploaded from anyone for free.We hope that we can improve this catalogue through your efforts,in the spirit of freedom and openness.*
+
+## (1) Brief Project Description
+
+*Novel-Wiki*
+
+## (2) Link to Project UI
+
+[Novel-Wiki](http://113.31.119.185)
+(Submission ID: 674639480)
+
+## (3) What does your application/UI do?
+
+*People can upload and retrieve works of fiction through this interface (for now).*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*MongoDB and mysql*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*People have access to all the data.*
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+*It depends on how enthusiastic people are about uploading,I'm guessing 500GB at least.*
+
+## (7) How are you structuring the data?
+
+*Meta data in our local serve on MongoDB and mysql,novels stored on filecoin.*
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+*We might have some compression before we put it in.*
+
+## (9)  What tech stack will you use to store the data?
+
+*Filecoin network and databse server.*
```

Comment:
Will you be using Powergate or Lotus/IPFS in order to support retrieval and storage on the filecoin network?

Reply:
lotus